apps/react-18-tests-v8/project.json

```diff
@@ -18,6 +18,14 @@
     "e2e:integration": {
       "dependsOn": [],
       "inputs": ["{projectRoot}/cypress-react-18.config.ts", "{workspaceRoot}/packages/react-examples/**/*.e2e.tsx?"]
+    },
+    "test:integration": {
+      "dependsOn": ["^build"],
+      "inputs": ["{projectRoot}/jest-react-18.config.js", "{workspaceRoot}/packages/react/**/*.(test|spec).tsx?"]
+    },
+    "build:integration": {
```

Comment:
add this target to invoke build on all dependants programatically as nx is  eating log files on CI